src/test/java/org/opensearch/neuralsearch/util/NeuralSearchClusterUtilTests.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.neuralsearch.util;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.opensearch.neuralsearch.util.NeuralSearchClusterTestUtils.mockClusterService;
+
+import org.opensearch.Version;
+import org.opensearch.cluster.service.ClusterService;
+import org.opensearch.test.OpenSearchTestCase;
+
+public class NeuralSearchClusterUtilTests extends OpenSearchTestCase {
+
+    public void testSingleNodeCluster() {
```

Comment:
try to put your tests function names like

test<WhatYourTesting>_when<ABCCondition>_then<XYZResult>